src/test/resources/network/brightspots/rcv/test_data/2017_minneapolis_mayor/2017_minneapolis_mayor_expected_MINNEAPOLIS_W-13_P-13_precinct_summary.csv

```diff
@@ -40,6 +40,6 @@ Active Ballots,953,,,952,,,943,,,939,,,866,,,789,,
 Current Round Threshold,52243,,,52200,,,50933,,,49874,,,46790,,,40838,,
 Inactive Ballots by Overvotes,0,,0,0,,0,0,,0,0,,0,0,,0,0,,0
 Inactive Ballots by Skipped Rankings,0,,0,0,,0,0,,0,0,,0,0,,0,0,,0
-Inactive Ballots by Exhausted Choices,0,,0,0,,0,0,,0,0,,0,0,,0,0,,0
+Inactive Ballots by Exhausted Choices,1,,1,2,,9,11,,4,15,,73,88,,77,165,,0
```

Comment:
How do I get a first round Exhausted ballot? If the only marking is an overvote and the overvote rule is lenient enough to not inactivate the ballot

Reply:
This is a funny consequence of EXHAUSTED encapsulating both FULLY_RANKED and PARTIALLY_RANKED.

Specifically, in this case, the first round is an "overvote" and the overvote decision is "skip to next rank". We mark these as exhausted:
```
            // If the final ranking is an overvote, even if we're trying to skip to the next rank,
            // we consider this inactive by exhausted choices -- not an overvote.
```

I don't think there's an obvious answer to what these should be marked as. EXHAUSTED seems good as any.

Reply:
I think exhausted is the right bucket for these. I wanted to make sure I documented this for when I'm looking back later and forget why I can have an exhausted ballot in Round 1